openedx/core/djangoapps/programs/tests/test_backpopulate_program_credentials.py

```diff
@@ -343,7 +343,6 @@ def test_handle_no_course_overview(self, mock_task, mock_get_programs):
         call_command('backpopulate_program_credentials', commit=True)
 
         mock_task.assert_called_once_with(self.alice.username)
-        mock_task.assert_not_called(self.bob.username)
```

Comment:
Didn't exist in 1.0.1, incorrect usage for the current version, and redundant after the previous assertion anyway.